### DISCUSSION TITLE
Proposal for a Sunset Policy

### DISCUSSION THREAD

**Discussion by jonasbn** [rojopolis/spellcheck-github-actions#94] [General]
I have written up [a proposal](https://dev.to/jonasbn/proposal-for-a-sunset-policy-for-a-github-action-2hbh) for a sunset policy for this GitHub action.

Any feedback would be most welcome.

I have published it on **dev.to**, since I also wanted to see if I could attract some more general feedback.

**Comment by facelessuser** [comment]
I have to admit, I'm surprised myself how much this action has grown. TBH, when I wrote pyspelling, I thought I'd be the only one using it :). Then this action was created on top of it and has grown in popularity.

It's always a double edge sword. Nice to provide something to the community that is well received, but it definitely adds s greater responsibility when when you start realize how many people depend on your project.

I'll take some time and look over the proposal and let you know my feelings at least.

Hats off to you for taking this on and helping to make spell checking even less painful.

**Reply by jonasbn** [reply]
Hi @facelessuser any feedback, ideas, suggestions etc. are most welcome.

**Reply by facelessuser** [reply]
I think it sounds reasonable. It sounds like the idea is to encourage people to use `vX`, etc., and only use `vX.X.X` when they absolutely need to with the goal of migrating back to `vX` when possible.  As you are giving people 365 days to utilize a `vX.X.X`, I think that gives more than sufficient time to fix a bad release and have people get back on the canonical train.
